product_variant_exclusion/models/product.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api, _
+from openerp.exceptions import ValidationError, UserError
+import psycopg2
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    variant_exclusion_ids = fields.One2many(
```

Comment:
We're happy with the copy functionality when duplicating a product template?  Personally I'm fine with it, just making sure its considered.